client/state/data-layer/wpcom/domains/validation-schemas/index.js

```diff
@@ -44,13 +43,19 @@ export const onSuccess = ( action, schemas ) => addValidationSchemas( schemas );
 /**
  * Create an error notice action when the request fails
  *
- * @param   {Object} action   Originating action (unused).
- * @param   {Object} error    Error information (unused).
- * @returns {Object}          error notice action
+ * @param   {Object} tlds   Originating action with the original list of requested tlds
+ * @param   {Object} error  Error information (query path, error message etc).
+ * @returns {[Action]}      An array of mc and tracks analytics events, one each per tld
  */
-export const onError = () =>
-	errorNotice(
-		translate( "We couldn't load the data for validating ccTLD specific requirements." )
+export const onError = ( { tlds }, error ) =>
+	flatten(
+		map( tlds, tld => [
```

Comment:
looks like you might find `composeAnalytics()` valuable
https://github.com/Automattic/wp-calypso/blob/master/client/state/analytics/actions.js#L36